Added the `output_data_type` parameter to convert the output data to ReadFile, and added ReadFileV2 (#21811)
demisto/content: Packs/CommonScripts/Scripts/ReadFile/README.md

| --- | --- |
| entryID | War room entryID of the file to read. |
| maxFileSize | Maximal file size to load, in bytes. Default is 1MB. |
| input_encoding | The character encoding of the file |
| output_data_type | The data type to which the output data is converted. |

> | output_meta_data | Set to true to output the meta data with the payload to `ReadFile` in the context, otherwise only the payload will be output to `FileData` in the context |

Reviewer: ```suggestion
| output_metadata | Set true in order to output additional metadata on the file, to context |
```
